storefront/yarn.lock

```diff
@@ -497,14 +497,7 @@
     fetch-event-stream "^0.1.5"
     qs "^6.12.1"
 
-"@medusajs/types@2.5.1":
-  version "2.5.1"
-  resolved "https://registry.yarnpkg.com/@medusajs/types/-/types-2.5.1.tgz#673322694815a3dc41fea1f8877ed903314fe6bf"
-  integrity sha512-46oH5mB2guFPcc/WXNzKu8yMWL4UIkkH2l3KWK61/VORYG8ebmODZ1q6KHBDbvlE4NdIrGyMf63FmYcKTMjb+w==
-  dependencies:
-    bignumber.js "^9.1.2"
-
-"@medusajs/types@^2.5.1":
+"@medusajs/types@2.6.0", "@medusajs/types@^2.5.1":
```

Comment:
@delicloredana please check what version of @medusajs/types we should use, it has to match @medusajs/medusa package version